docs/01-debugging.md

```diff
@@ -0,0 +1,25 @@
+# Debugging
+For debugging install [react-native-debugger](https://github.com/jhen0409/react-native-debugger)
+
+## Setup connectivity from device to metro
+On Android you have 2 options:
+* Setup a tunnel though adb: `adb reverse tcp:8081 tcp:8081`
+* Open the development options of the react-native app (shake device or Ctrl+M) and set a development host
```

Comment:
Third:
* Attach the device with a cable   
:smile:

Reply:
Just attaching it is not enough. You need to run adb reverse.

Reply:
Maybe I don't quite understand what you mean by connectivity. I never executed any of the two options you listed.

Reply:
Yep, connectivity is not a good word I guess... I have never used adb reverse as well, and was never asked for a development host (whatever that is).

Reply:
Using the correct term: reverse tunnel now